### PR TITLE
finished 0x06b

### PR DESCRIPTION
**Pull request by sushi2k**
finalising the iOS basic security testing chapter

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit ba6d718a9630366736d576d5e937dedf0d1966f5: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -320,6 +304,22 @@ $ ios plist cat <myfile.plist>
...
+
+[Passionfruit](https://github.com/chaitin/passionfruit/ "Passionfruit") is an iOS app blackbox assessment tool that is using the Frida server on the iOS device and visualizes many standard tasks via Vue.js. It can be installed with npm.
+
+```shell
+$ npm install -g passionfruit
+$ passionfruit
+listening on http://localhost:31337
+```
+
+Once you executed the command `passionfruit` a local server will be started on port 31337. Connect your jailbroken device with the Frida server running, or a non-jailbroken device with a repackaged app including Frida to your macOS device via USB. Once you click on the "iPhone" icon in the example below you will get an overview of all installed apps.
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: Once you executed the command  -> When you execute the command `passionfruit`,

**Comment by OMTGreleaser** [comment]
Results for commit b36da0978a5fabbd2887cdf0a2868afa65942299: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit c0092277d62839f710b51b7f0f2df1e3bee8e401: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**